c7n/resources/rds.py

```diff
@@ -274,6 +274,133 @@ def process(self, dbs):
                 **params)
 
 
+@filters.register('minor-upgrade-flag')
+class NoMinorUpgradeFlag(Filter):
```

Comment:
we don't want additional filters that can be implemented as simple value filters.